server/routerlicious/packages/routerlicious/src/scribe/index.ts

```diff
@@ -46,16 +46,18 @@ export async function scribeCreate(config: Provider): Promise<IPartitionLambdaFa
         client.collection<ISequencedOperationMessage>(messagesCollectionName),
     ]);
 
-    await scribeDeltas.createIndex(
-        {
-            "documentId": 1,
-            "operation.sequenceNumber": 1,
-            "tenantId": 1,
-        },
-        true);
-
     if (createCosmosDBIndexes) {
+        await scribeDeltas.createIndex({ documentId: 1 }, false);
+        await scribeDeltas.createIndex({ tenantId: 1 }, false);
```

Comment:
Is there any implication or unexpected side effect now that we have to set `unique` to `false`?

Reply:
Actually, this is something I was thinking too. Now that we have document id as a guid, we could make the docid uniqueness set to true. @tanviraumi thoughts?

Reply:
"Unique" is probably per entry level? So each document will have multiple entries (one per operation) with the same documentId.